core/src/main/scala/com/snowplowanalytics/snowplow/eventgen/protocol/event/EventType.scala

```diff
@@ -19,12 +19,13 @@ case class EventFrequencies(
   unstruct: Int,
   pageView: Int,
   pagePing: Int,
+  remainingFields: Int,
```

Comment:
If we add new field here, do we also need default [config](https://github.com/snowplow-incubator/snowplow-event-generator/blob/b9000d1f0e355684f8235eb1a3736615f7555dd5/sinks/src/main/resources/application.conf#L22) for it?

Reply:
Yep, looks like it. Thanks for the heads up!